mysql/set.go

```diff
@@ -57,7 +57,7 @@ func ParseSetName(elems []string, name string) (Set, error) {
 		if _, ok := marked[key]; ok {
 			value |= (1 << uint64(i))
 			delete(marked, key)
-			items = append(items, key)
```

Comment:
It is covered by old test case which inserted a set with uppercase string, then get back lowercase string back.